services/sidecar/requirements/_test.in

```diff
@@ -13,6 +13,7 @@ pytest-cov
 pytest-instafail
 pytest-mock
 pytest-sugar
+pytest-lazy-fixture
```

Comment:
By adding a new requirements, then ``make reqs`` will force an upgrade of ALL libraries
I personally like to separate the "general" upgrades from new additions. Two commits/steps:

-  ``make reqs upgrade=pytst-lazy-fixture``: will ONLY upgrade a specific library. Now you can commit
- ``make reqs`` updates all